fe/fe-core/src/main/java/com/starrocks/lake/StarMgrMetaSyncer.java

```diff
@@ -107,7 +108,12 @@ public static void dropTabletAndDeleteShard(ComputeResource computeResource,
             try {
                 if (isFileBundling) {
                     if (pickBackendId == -1) {
-                        pickBackendId = LakeAggregator.chooseAggregatorNode(computeResource).getId();
+                        ComputeNode cn = LakeAggregator.chooseAggregatorNode(computeResource);
+                        if (cn == null) {
+                            LOG.error("No alive compute node for aggregate operation.");
```

Comment:
this is a background task check and run repeatedly, reduce the log level and evaluate the possibility of logs when there is no node in the warehouse, how many logs may be printed and possibly need to sample the logs